products/atv_base.mk

```diff
@@ -108,3 +107,6 @@ $(call inherit-product-if-exists, external/naver-fonts/fonts.mk)
 $(call inherit-product-if-exists, frameworks/base/data/keyboards/keyboards.mk)
 $(call inherit-product-if-exists, frameworks/webview/chromium/chromium.mk)
 $(call inherit-product, $(SRC_TARGET_DIR)/product/core_minimal.mk)
+# for b2g
+$(call inherit-product, external/moztt/fonts.mk)
```

Comment:
Because google include tv necessary stuff in atv_base.mk. they don't inherit generic_no_telephony.mk